src/constants.ts

```diff
@@ -91,7 +91,3 @@ export const SIDECAR_OUTPUT_CHANNEL: LogOutputChannel = window.createOutputChann
   "Confluent (Sidecar)",
   { log: true },
 );
-
-/** Flink Config settings names */
-export const FLINK_CONFIG_COMPUTE_POOL = "confluent.flink.computePoolId";
-export const FLINK_CONFIG_DATABASE = "confluent.flink.database";
```

Comment:
Moved over to `src/preferences/constants.ts`: https://github.com/confluentinc/vscode/blob/6b1e5113f92f1901623214432e432bd9844a16fb/src/preferences/constants.ts#L43-L46